test/hga/test_utilities.py

```diff
@@ -0,0 +1,250 @@
+from os import makedirs
```

Comment:
I've consolidated the testing into some utility functions in this file. Previously, the tests were run by:
1) Reading information from some files specific to each collection (e.g., `avnir_info.py`).
2) Identifying the test expected results specific test (from the `avnir_info.py` file)/
3) Writing a new file that would contain `pytest` compatible test functions.
3) Running `pytest` on the written file.

Instead, the `check_request_output` function in this file just tries to assert the output matches the expected status. If not, an `AssertionError` will be raised. I've added the optional error message argument to these assertions, so that the failed cell will print something informative to someone reviewing the notebook.